src/controller/domain_handle.rs

```diff
@@ -209,6 +225,22 @@ impl DomainHandle {
         Ok(())
     }
 
+    pub(super) fn send_to_healthy(
```

Comment:
We should be more principled about sending to possibly failed nodes. At very least we should probably also use this function so we never send to nodes that we know are failed...

Reply:
Agreed. I've marked the old `send` it as deprecated in 9796065, but got no compile warnings -- so I suspect it's already unused? (though that should also have produced warnings)

Reply:
Never mind, I was being stupid (ae832ca). It's called in lots of places.